markdown/3.0/en/documentation/install-dist.md

```diff
@@ -3,34 +3,25 @@
 ---
 Depending on whether you are using [Node.js](https://nodejs.org/), you have the following two ways of installing FACT-Finder Web Components.
 
-### Using Node.js/Bower
+### Using Node.js/bower or Node.js/npm
 
 ---
-1. Install **Node.js** [(Download)](https://nodejs.org/en/download/)
-2. Install **Bower** -> `npm install bower -g`
-3. Create a **bower.json** file -> `bower init`
-4. Add FACT-Finder Web Components as a dependency to the **bower.json** so it looks somewhat like this:
-    ```json
-    {
-      "name": "your-project",
-      "description": "",
-      "main": "index.js",
-      "dependencies": {
-        "ff-web-components": "https://github.com/FACT-Finder-Web-Components/ff-web-components.git#1.2.11"
-      },
-      "license": "ISC",
-      "homepage": ""
-    }
-    ```
-5. From within the directory the **bower.json** file is located run `bower install`.
-
-**Note:** We have added a `package.json` file as well so you can use `npm` or `yarn` instead of Bower.
+Add FACT-Finder Web Components as a dependency to the **bower.json** or **package.json**:
+```json
+{
+  "dependencies": {
+    "ff-web-components": "https://github.com/FACT-Finder-Web-Components/ff-web-components.git#3.0.0"
+  }
+}
+```
 
 
 ### Manual Installation
 
 ---
-1. Download (or clone) the demo project from the [Github Release Page](https://github.com/FACT-Finder-Web-Components/ff-web-components/releases)
+1. Download the asset of your required version from [Github Release Page](https://github.com/FACT-Finder-Web-Components/ff-web-components/releases)
 2. Unzip the file
-3. We have included a zip file called `bower_components.zip`. This file is actually a zipped directory of the **Using Node.js/Bower** approach. Unzip the `bower_components.zip` file
-4. Copy the bower_components directory or its contents to your project
+3. We have included a zip file called `dist.zip`. This file is actually a zipped directory of the **Using Node.js/Bower** approach. Unzip the `dist.zip` file
```

Comment:
```suggestion
3. We have included `dist.zip`, a zip file containing the files used in the **Using Node.js/Bower** approach. Unzip the `dist.zip` file
```

Reply:
Rewrote section as we are not including a `dist.zip`